### ISSUE TITLE
Crash on FDMessageController updateBottomViewWith:andHeight:

### ISSUE THREAD

**Issue by lyubimovm** [freshdesk/freshchat-ios#16]
There is a crash reports pointing to freshchat-ios library
`Fatal Exception: NSInvalidArgumentException
*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]`

```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x1816aed8c __exceptionPreprocess
1  libobjc.A.dylib                0x1808685ec objc_exception_throw
2  CoreFoundation                 0x181647750 _CFArgv
3  CoreFoundation                 0x1815800cc -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]
4  CoreFoundation                 0x18157ff48 +[NSDictionary dictionaryWithObjects:forKeys:count:]
5  MYAPPNAME                        0x1004d06bc -[FDMessageController updateBottomViewWith:andHeight:]
6  MYAPPNAME                        0x1004d58ec __41-[FDMessageController processPendingCSAT]_block_invoke
7  libdispatch.dylib              0x180fa0aa0 _dispatch_call_block_and_release
8  libdispatch.dylib              0x180fa0a60 _dispatch_client_callout
9  libdispatch.dylib              0x180fad65c _dispatch_main_queue_callback_4CF$VARIANT$mp
10 CoreFoundation                 0x181657070 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
11 CoreFoundation                 0x181654bc8 __CFRunLoopRun
12 CoreFoundation                 0x181574da8 CFRunLoopRunSpecific
13 GraphicsServices               0x183559020 GSEventRunModal
14 UIKit                          0x18b591758 UIApplicationMain
15 MYAPPNAME                        0x1002dd1f8 main (AppDelegate.swift:29)
16 libdyld.dylib                  0x181005fc0 start
```
Unfortunately I could not reproduce the problem. I only see the periodic crashes in Crashlytics.
Freshchat iOS SDK v1.4.1 and earlier
Crashes occurs in any versions of iOS

**Comment by SanjithKanagavel** [comment]
@lyubimovm. Thanks for sharing this. We will investigate on this. 

Please share the following details : 
1. Share us the full **Crashlytics** crash log to support@freshchat.com
2. Steps to reproduce crash if you were able to do it.

**Comment by lyubimovm** [comment]
1. Full crash log sent to support email
2. The problem was observed in all versions over the last 3 months. The latest version of freshchat-ios library which is in the production - 1.4.1
3. Unfortunately STR was not found

**Comment by SanjithKanagavel** [comment]
@lyubimovm. Thanks for sharing us the details. We will try to investigate on this issue and fix them.

**Comment by SanjithKanagavel** [comment]
@lyubimovm we have fixed this issue in v1.4.6. Thanks for letting us know this.
